java destination on RHEL6 (CentOS6)

When I try to compile 0.4.1 on CentOS6 with OpenJDK 1.7, I get the following error during compilation:
```
  CC       modules/java/modules_java_libmod_java_la-java-grammar.lo
  CC       modules/java/modules_java_libmod_java_la-java-plugin.lo
  CC       modules/java/modules_java_libmod_java_la-java-destination.lo
  CC       modules/java/modules_java_libmod_java_la-java-destination-proxy.lo
modules/java/java-destination-proxy.c:59: error: redefinition of typedef 'JavaDestinationProxy'
modules/java/java-destination-proxy.h:30: note: previous declaration of 'JavaDestinationProxy' was here
make[1]: Leaving directory `/builddir/build/BUILD/syslog-ng-incubator-0.4.1'
make[1]: *** [modules/java/modules_java_libmod_java_la-java-destination-proxy.lo] Error 1
```

--- a/modules/java/java-destination-proxy.c
+++ b/modules/java/java-destination-proxy.c
@@ -42,7 +42,7 @@ typedef struct _JavaDestinationImpl
   jmethodID mi_flush;
 } JavaDestinationImpl;
 
-typedef struct _JavaDestinationProxy
+struct _JavaDestinationProxy
 {
   jclass loaded_class;
   JavaDestinationImpl dest_impl;
@@ -56,7 +56,7 @@ typedef struct _JavaDestinationProxy
   jmethodID loader_constructor;
   jmethodID mi_loadclass;
 
-} JavaDestinationProxy;
+};
 
 
 static gboolean

